protocol-specs/generators/lisk_codec/proto_files/object.proto

```diff
@@ -0,0 +1,19 @@
+syntax = 'proto2';
+
+message Asset {
+	required string data = 1;
+}
+
+message Objects {
+	required bytes address = 1;
+	required uint64 balance = 2;
+	required bool isDelegate = 3;
+	required string name = 4;
+	required Asset asset = 5;
+}
+
+message ObjectOptionalProp {
+	required bool isActive = 1;
+	optional bytes data = 2;
+	required uint64 value = 3;
+}
```

Comment:
Better to rename it to `ObjectWithOptionalProp` as its mixed of required and optional fields.

Reply:
Done